docs/kubernetes_quickstart.md

```diff
@@ -0,0 +1,69 @@
+# Kubernetes Helm Chart for Freqtrade 
+
+This helm chart is like Freqtrade for educational purposes. Any professional use is not recommended. Run Freqtrade in dry-run (see configs).
+Please check the logs after deployment for 
+
+```
+freqtrade.exchange.exchange - INFO - Instance is running with dry_run enabled
+```
+
+## Prerequistes
+
+Look for the appropriate latest version of your preferred OS. Ask ChatGPT/Google to help you.
```

Comment:
we will NOT recommend the use of chatGPT in the docs. 

it's answers are flaky at best and strongly depend on the actual question asked.